index.html

```diff
@@ -214,11 +211,11 @@ <h4 ><span>Wynand</span> Badenhorst</h4>
         <!-- Co-VP of Outreach -->
         <div class="col-md-3">
                 <div class="exec_img">
-                    <img src="images/harman.jpg" alt="Harmon" />
+                    <img src="images/harman.jpg" alt="Harman" />
+                    <span class="title">Co-VP of Outreach</span>
```

Comment:
The span is moved inside the div. It belongs outside. Otherwise nice catch on the spelling